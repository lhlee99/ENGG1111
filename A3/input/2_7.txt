aa  aa  aaa a aa
aaaa
