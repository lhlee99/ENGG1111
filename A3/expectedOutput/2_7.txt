0 5
1 8
4 9
5 10
8 12
9 14
10 15
